idaes/core/util/unit_costing.py

```diff
@@ -262,13 +291,14 @@ def pressure_changer_costing(self, Mat_factor="stain_steel",
              work_mechanical[self.parent_block().
                              flowsheet().config.time.first()])
 
-    work_hp = pyunits.convert(w, to_units=pyunits.hp)
+    work_hp = pyunits.convert(w, to_units=pyunits.hp) / self.number_of_units
 
     # if compressor is == False, that means pressure changer is a Turbine
     if self.parent_block().config.compressor is False:
         #                           -1*work_hp because work is negative
         def CP_rule(self):
-            return self.purchase_cost == 530*(-1/pyunits.hp * work_hp)**0.81
+            return self.purchase_cost == self.number_of_units \
+                * 530*(-1/pyunits.hp * work_hp)**0.81
```

Comment:
A couple of minor formatting notes (that don't need to be fixed now):

1. PEP* standards say the operator should go before a line break rather than after, i.e.:

```
a * \
b
```

rather than 

```
a \
* b
```

2. Parenthesis tend to make things look neater and more readable than the linebreak character `\`.